src/widget/widget.ts

```diff
@@ -19,6 +19,7 @@ const html = String.raw;
 
 const MODULE_VERSION = process.env.MODULE_VERSION;
 const COMPRESSED_ORG = process.env.COMPRESSED_ORG;
+const useDefaultButtonStyle = COMPRESSED_ORG !== 'GYJwhgtkA' ? true : false;
```

Comment:
Not a big fan of this 🤔 Are we sure that there are no other options to this, then having an org-specific class?

Reply:
I am nota big fan of this approach either, but the style.buttonLightOutline should only be applied when the organization is set to FRAM to make the widget and the web travel planner appear similar. I am not aware of another approach, except maybe reorganising the fram colours to achieve the same style. But I am sure that is a lot of work, and will also lead to other unwanted changes in the styling🤷‍♂️

Reply:
Not sure, tbh. How about using `getOrgData()`, is that an option?